app/models/user.rb

```diff
@@ -38,7 +38,7 @@ class User < ActiveRecord::Base
   def role?(base_role)
     return false unless role
 
-    ROLES.index(base_role) <= ROLES.index(role.to_sym)
+    ROLES.index(base_role) == ROLES.index(role.to_sym)
```

Comment:
This looks like it could be simplified to `base_role.to_sym == role.to_sym`. The `ROLES` variable could either be completely removed or instead used to verify that base_role is valid `[PSEUDOCODE] unless base_role is part of ROLES throw some error`.